crates/biome_fs/src/path.rs

```diff
@@ -54,11 +52,7 @@ pub struct BiomePath {
 impl BiomePath {
     pub fn new(path_to_file: impl Into<Utf8PathBuf>) -> Self {
         let path = path_to_file.into();
-        let kind = if path.is_dir() {
-            FileKinds::Directory
-        } else {
-            path.file_name().map(Self::priority).unwrap_or_default()
-        };
```

Comment:
This is introduced in https://github.com/biomejs/biome/commit/43092347c3815179316688b7cbca58802ad444fd, but it seems not used. `Utf8Path::is_dir()` requires an I/O syscall, that is relatively heavy when it's called many times.

Reply:
I've been bothered by that one for a while, and wished to remove it too. Glad you did!